### DISCUSSION TITLE
NoIP no longer working after latest update (v2.6.0  c338c28)

### DISCUSSION THREAD

**Discussion by Rhostigma** [qdm12/ddns-updater#627] [Q&A] [answered]
![image](https://github.com/qdm12/ddns-updater/assets/158821610/53d1aba7-504c-447a-8a05-2888eb630c06)

```json
{
  "settings": [
    {
      "provider": "noip",
      "domain": "filler",
      "host": "filler",
      "username": "lorumipsum",
      "password": "lorumipsum",
      "ip_version": "ipv4",
      "ipv6_suffix": "",
      "provider_ip": true
    }
  ]
}
```

using this in my config.json

**Comment by qdm12** [comment, marked as answer]
Since commit c49466297635303ee6a37a2c6605cca3df6888ec the public IP echo service noip was removed because it would only support http and not https, so a malicious actor could set a bad ip address and thus mess up your DNS records which would be quite bad. No report of this yet, but better to prevent it! So just change `PUBLICIPV4_HTTP_PROVIDERS=noip` to for example `PUBLICIPV4_HTTP_PROVIDERS=all` (or remove it so it defaults to `all`)

Note however the noip "registrar" (known as provider for now, some renaming to do soon) is still supported 😉 It's just those errors like `unknown provider` were quite generic, it's changed in 133956f0829e3ecdeacd7151676fa607fde1e860 (in the latest image).

**Reply by Rhostigma** [reply]
Thank you!
